src/get-section.c

```diff
@@ -64,7 +64,7 @@ int main(int argc, char ** argv) {
         ABORT(255);
     }
 
-    singularity_message(DEBUG, "Iterating through file looking for sections matching: \%%s\n", section);
+    singularity_message(DEBUG, "Iterating through file looking for sections matching: %s\n", section);
```

Comment:
Ah - so that's what it was trying to attempt!  (As written, it was an invalid format specifier.)  In that case, it'll need to change to `%%%s`.